Could not install with native-comp

<!-- To help us debug your issue, please complete these sections: -->

Please make sure to follow these steps (and mark the checkboxes):

- [x] run `brew update` and try to reproduce the issue again
- [x] run `brew doctor`, fix all issues and try to reproduce your issue again
- [x] run `brew config` and `brew doctor` and include their output

## What you were trying to do

Install emacs-plus@28 with native comp.

## What happened (include command output)

The `./configure` step failed, complaining about not being able to find zlib. Adding `depends_on "zlib" => :build` to the appropriate section of the formula allowed the configure step to complete. The log below is from prior to adding this line to the formula.

<pre>
$ brew install d12frosted/emacs-plus/emacs-plus@28 --with-native-comp --with-nobu417-big-sur-icon  --with-no-frame-refocus
</pre>

<details>
  <summary>Command output</summary>
  <pre>
Updating Homebrew...
==> Auto-updated Homebrew!
Updated 1 tap (homebrew/core).
==> Updated Formulae
Updated 4 formulae.

==> Installing emacs-plus@28 from d12frosted/emacs-plus
==> Cloning https://github.com/emacs-mirror/emacs.git
Updating /Users/ryan/Library/Caches/Homebrew/emacs-plus@28--git
==> Checking out branch master
Already on 'master'
Your branch is up to date with 'origin/master'.
HEAD is now at f4dc646e0d * lisp/international/mule-cmds.el (mule--ucs-names-group): Better char check.
Warning: A newer Command Line Tools release is available.
Update them from Software Update in System Preferences or run:
  softwareupdate --all --install --force

If that doesn't show you any updates, run:
  sudo rm -rf /Library/Developer/CommandLineTools
  sudo xcode-select --install

Alternatively, manually download them from:
  https://developer.apple.com/download/more/.
You should download the Command Line Tools for Xcode 12.5.

==> Patching
==> Applying no-frame-refocus-cocoa.patch
patching file src/frame.c
Hunk #1 succeeded at 2096 (offset 40 lines).
==> Applying fix-window-role.patch
patching file src/nsterm.m
Hunk #1 succeeded at 8881 (offset 113 lines).
==> Applying system-appearance.patch
patching file src/frame.h
patching file src/nsfns.m
Hunk #1 succeeded at 1260 (offset 7 lines).
patching file src/nsterm.m
Hunk #1 succeeded at 2173 (offset -21 lines).
Hunk #2 succeeded at 5669 (offset -82 lines).
Hunk #3 succeeded at 5915 (offset -82 lines).
Hunk #4 succeeded at 5965 (offset -82 lines).
Hunk #5 succeeded at 6191 (offset -82 lines).
Hunk #6 succeeded at 9212 (offset 80 lines).
Hunk #7 succeeded at 10298 (offset 284 lines).
==> ImageMagick PKG_CONFIG_PATH:
/usr/local/opt/imagemagick/lib/pkgconfig
==> ./autogen.sh
==> ./configure --enable-locallisppath=/usr/local/share/emacs/site-lisp --infodir=/usr/local/Cellar/emacs-plus@28/28.0.50/share/info/emacs --prefix=/usr/local/Cellar/emacs-plus@28/2
Last 15 lines from /Users/ryan/Library/Logs/Homebrew/emacs-plus@28/02.configure:
checking for MagickAutoOrientImage... yes
checking for getaddrinfo_a in -lanl... no
checking for lgetfilecon in -lselinux... no
checking for gnutls >= 2.12.2... yes
checking for libsystemd >= 222... no
checking for jansson >= 2.7... yes
checking for sys/inotify.h... no
checking for libkqueue... no
checking for library containing kqueue... none required
checking for jpeglib 6b or later... -ljpeg
checking for lcms2... yes
checking for library containing inflateEnd... no
checking for dladdr... yes
checking for dlfunc... no
configure: error: '--with-nativecomp' requires zlib

If reporting this issue please do so at (not Homebrew/brew or Homebrew/core):
  https://github.com/d12frosted/homebrew-emacs-plus/issues

These open issues may also help:
Permission issues with emacs-plus running as a service https://github.com/d12frosted/homebrew-emacs-plus/issues/304

Error: A newer Command Line Tools release is available.
Update them from Software Update in System Preferences or run:
  softwareupdate --all --install --force

If that doesn't show you any updates, run:
  sudo rm -rf /Library/Developer/CommandLineTools
  sudo xcode-select --install

Alternatively, manually download them from:
  https://developer.apple.com/download/more/.
You should download the Command Line Tools for Xcode 12.5.

ryan@urania Movies/Mobile Videos »   sudo rm -rf /Library/Developer/CommandLineTools                                                                                             1 ↵
Password:
  sudo xcode-select --install
ryan@urania Movies/Mobile Videos »   sudo xcode-select --install
xcode-select: note: install requested for command line developer tools
ryan@urania Movies/Mobile Videos » brew install d12frosted/emacs-plus/emacs-plus@28 --with-native-comp --with-nobu417-big-sur-icon  --with-no-frame-refocus
Updating Homebrew...
==> Auto-updated Homebrew!
Updated 1 tap (homebrew/core).
==> Updated Formulae
Updated 7 formulae.

==> Installing emacs-plus@28 from d12frosted/emacs-plus
==> Cloning https://github.com/emacs-mirror/emacs.git
Updating /Users/ryan/Library/Caches/Homebrew/emacs-plus@28--git
==> Checking out branch master
Already on 'master'
2021-05-25 19:25:45 -0400

./configure
--disable-dependency-tracking
--disable-silent-rules
--enable-locallisppath=/usr/local/share/emacs/site-lisp
--infodir=/usr/local/Cellar/emacs-plus@28/28.0.50/share/info/emacs
--prefix=/usr/local/Cellar/emacs-plus@28/28.0.50
--with-xml2
--with-gnutls
--with-native-compilation
--without-dbus
--with-imagemagick
--with-modules
--with-rsvg
--with-ns
--disable-ns-self-contained

configure: WARNING: unrecognized options: --disable-dependency-tracking
checking for xcrun... xcrun
checking for make... yes
checking for GNU Make... make
checking build system type... x86_64-apple-darwin20.5.0
checking host system type... x86_64-apple-darwin20.5.0
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables...
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether the compiler supports GNU C... yes
checking whether clang accepts -g... yes
checking for clang option to enable C11 features... none needed
checking whether the compiler is clang... yes
checking for compiler option needed when checking for declarations... -Werror=implicit-function-declaration
checking for ar... ar
checking whether clang and cc understand -c and -o together... yes
checking for stdio.h... yes
checking for stdlib.h... yes
checking for string.h... yes
checking for inttypes.h... yes
checking for stdint.h... yes
checking for strings.h... yes
checking for sys/stat.h... yes
checking for sys/types.h... yes
checking for unistd.h... yes
checking for wchar.h... yes
checking for minix/config.h... no
checking for linux/fs.h... no
checking for malloc.h... no
checking for sys/systeminfo.h... no
checking for sys/sysinfo.h... no
checking for coff.h... no
checking for pty.h... no
checking for sys/resource.h... yes
checking for sys/utsname.h... yes
checking for pwd.h... yes
checking for utmp.h... yes
checking for util.h... yes
checking for sanitizer/lsan_interface.h... yes
checking for sys/socket.h... yes
checking for sys/param.h... yes
checking for pthread.h... yes
checking for malloc/malloc.h... yes
checking for sys/un.h... yes
checking for vfork.h... no
checking for dirent.h... yes
checking for execinfo.h... yes
checking for stdio_ext.h... no
checking for sys/vfs.h... no
checking for sys/fs_types.h... no
checking for getopt.h... yes
checking for sys/cdefs.h... yes
checking for sys/time.h... yes
checking for ieee754.h... no
checking for limits.h... yes
checking for sys/select.h... yes
checking for sys/random.h... yes
checking whether it is safe to define __EXTENSIONS__... yes
checking whether _XOPEN_SOURCE should be defined... no
checking how to run the C preprocessor... clang -E
checking for grep that handles long lines and -e... /usr/bin/grep
checking for egrep... /usr/bin/grep -E
checking for Minix Amsterdam compiler... no
checking for ranlib... ranlib
checking for special C compiler options needed for large files... no
checking for _FILE_OFFSET_BITS value needed for large files... no
checking whether clang accepts -O... yes
checking whether the compiler is clang... yes
checking whether C compiler handles -Werror -Wunknown-warning-option... yes
checking whether C compiler handles -Wno-switch... yes
checking whether C compiler handles -Wno-pointer-sign... yes
checking whether C compiler handles -Wno-string-plus-int... yes
checking whether C compiler handles -Wno-unknown-attributes... yes
checking whether C compiler handles -Wno-initializer-overrides... yes
checking whether C compiler handles -Wno-tautological-compare... yes
checking whether C compiler handles -Wno-tautological-constant-out-of-range-compare... yes
checking for a BSD-compatible install... /usr/bin/install -c
checking command to symlink files in the same directory... ln -s
checking for install-info... /usr/local/opt/texinfo/bin/install-info
checking for gzip... /usr/bin/gzip
checking for 'find' args to delete a file... -delete
checking for brew... no
checking for -znocombreloc... not needed
checking whether addresses are sanitized... no
checking for library containing sqrt... none required
checking for pkg-config... /usr/local/Homebrew/Library/Homebrew/shims/mac/super/pkg-config
checking pkg-config is at least version 0.9.0... yes
checking for machine/soundcard.h... no
checking for sys/soundcard.h... no
checking for soundcard.h... no
checking for mmsystem.h... no
checking for _oss_ioctl in -lossaudio... no
checking for alsa >= 1.0.0... no
checking for ADDR_NO_RANDOMIZE... no
checking for sys/wait.h that is POSIX.1 compatible... yes
checking for net/if.h... yes
checking for ifaddrs.h... yes
checking for net/if_dl.h... yes
checking for struct ifreq.ifr_flags... yes
checking for struct ifreq.ifr_hwaddr... no
checking for struct ifreq.ifr_netmask... no
checking for struct ifreq.ifr_broadaddr... yes
checking for struct ifreq.ifr_addr... yes
checking for struct ifreq.ifr_addr.sa_len... yes
checking whether gcc understands -MMD -MF... yes
checking for X... libraries , headers
checking for AppKit/AppKit.h... yes
checking for Mac OS X 10.6 or newer... yes
checking whether the compiler supports GNU Objective C... yes
checking whether clang accepts -g... yes
checking if the Objective C compiler supports instancetype... yes
checking if the Objective C compiler defaults to C99... yes
checking whether malloc is Doug Lea style... no
checking for sbrk... yes
checking for getpagesize... yes
checking for __lsan_ignore_object... no
checking for fork... yes
checking for vfork... yes
checking for fchmod... yes
checking for canonicalize_file_name... no
checking for faccessat... yes
checking for realpath... yes
checking for lstat... yes
checking for readlinkat... yes
checking for explicit_bzero... no
checking for memset_s... yes
checking for fchmodat... yes
checking for lchmod... yes
checking for fcntl... yes
checking for fdopendir... yes
checking for fstatat... yes
checking for fsync... yes
checking for futimens... yes
checking for getrandom... no
checking for gettimeofday... yes
checking for mkostemp... yes
checking for pipe2... no
checking for pselect... yes
checking for pthread_sigmask... yes
checking for readlink... yes
checking for isblank... yes
checking for iswctype... yes
checking for strtoimax... yes
checking for symlink... yes
checking for localtime_r... yes
checking for timegm... yes
checking for utimensat... yes
checking for getdtablesize... yes
checking for futimes... yes
checking for futimesat... no
checking for lutimes... yes
checking for working mmap... yes
checking for main in -lXbsd... no
checking for pthread library... none needed
checking for thread support... yes
checking for librsvg-2.0 >= 2.14.0... yes
checking for MagickWand >= 7... yes
checking for MagickRelinquishMemory... yes
checking for MagickExportImagePixels... yes
checking for MagickMergeImageLayers... yes
checking for MagickAutoOrientImage... yes
checking for getaddrinfo_a in -lanl... no
checking for lgetfilecon in -lselinux... no
checking for gnutls >= 2.12.2... yes
checking for libsystemd >= 222... no
checking for jansson >= 2.7... yes
checking for sys/inotify.h... no
checking for libkqueue... no
checking for library containing kqueue... none required
checking for jpeglib 6b or later... -ljpeg
checking for lcms2... yes
checking for library containing inflateEnd... no
checking for dladdr... yes
checking for dlfunc... no
configure: error: '--with-nativecomp' requires zlib

HOMEBREW_VERSION: 3.1.9-8-g6f1398a
ORIGIN: https://github.com/Homebrew/brew
HEAD: 6f1398a5338d851151041c81562da38b62cceff3
Last commit: 8 hours ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: 854b4eb2ff7f2fcbb28d0f69cf5c6d6b5d04dd21
Core tap last commit: 15 minutes ago
Core tap branch: master
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CASK_OPTS: []
HOMEBREW_DISPLAY: /private/tmp/com.apple.launchd.pQYcmh7dIC/org.xquartz:0
HOMEBREW_EDITOR: vim
HOMEBREW_MAKE_JOBS: 4
HOMEBREW_NO_BOOTSNAP: set
Homebrew Ruby: 2.6.3 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: quad-core 64-bit kabylake
Clang: 12.0.5 build 1205
Git: 2.31.1 => /usr/local/bin/git
Curl: 7.64.1 => /usr/bin/curl
macOS: 11.4-x86_64
CLT: 12.5.0.0.1.1617976050
Xcode: 12.5

HOMEBREW_CC: clang
HOMEBREW_CXX: clang++
CFLAGS: -I/usr/local/opt/gcc/include -I/usr/local/opt/libgccjit/include -I/usr/local/opt/gmp/include -I/usr/local/opt/jpeg/include
LDFLAGS: -L/usr/local/lib/gcc/11 -I/usr/local/opt/gcc/include -I/usr/local/opt/libgccjit/include -I/usr/local/opt/gmp/include -I/usr/local/opt/jpeg/include
MAKEFLAGS: -j4
CMAKE_PREFIX_PATH: /usr/local/opt/texinfo:/usr/local/opt/libffi:/usr/local/opt/m4:/usr/local/opt/readline:/usr/local/opt/openssl@1.1:/usr/local/opt/sqlite:/usr/local/opt/icu4c:/usr/local/opt/gnu-getopt:/usr/local
CMAKE_INCLUDE_PATH: /Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/System/Library/Frameworks/OpenGL.framework/Versions/Current/Headers
CMAKE_LIBRARY_PATH: /Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/System/Library/Frameworks/OpenGL.framework/Versions/Current/Libraries
CMAKE_FRAMEWORK_PATH: /usr/local/opt/python@3.9/Frameworks
PKG_CONFIG_PATH: /usr/local/opt/imagemagick/lib/pkgconfig:/usr/local/opt/gmp/lib/pkgconfig:/usr/local/opt/bdw-gc/lib/pkgconfig:/usr/local/opt/libffi/lib/pkgconfig:/usr/local/opt/readline/lib/pkgconfig:/usr/local/opt/guile/lib/pkgconfig:/usr/local/opt/libidn2/lib/pkgconfig:/usr/local/opt/libtasn1/lib/pkgconfig:/usr/local/opt/nettle/lib/pkgconfig:/usr/local/opt/p11-kit/lib/pkgconfig:/usr/local/opt/openssl@1.1/lib/pkgconfig:/usr/local/opt/libevent/lib/pkgconfig:/usr/local/opt/c-ares/lib/pkgconfig:/usr/local/opt/jemalloc/lib/pkgconfig:/usr/local/opt/nghttp2/lib/pkgconfig:/usr/local/opt/unbound/lib/pkgconfig:/usr/local/opt/gnutls/lib/pkgconfig:/usr/local/opt/libpng/lib/pkgconfig:/usr/local/opt/freetype/lib/pkgconfig:/usr/local/opt/fontconfig/lib/pkgconfig:/usr/local/opt/pcre/lib/pkgconfig:/usr/local/opt/sqlite/lib/pkgconfig:/usr/local/opt/xz/lib/pkgconfig:/usr/local/opt/python@3.9/lib/pkgconfig:/usr/local/opt/glib/lib/pkgconfig:/usr/local/opt/libpthread-stubs/lib/pkgconfig:/usr/local/opt/libxau/lib/pkgconfig:/usr/local/opt/libxdmcp/lib/pkgconfig:/usr/local/opt/libxcb/lib/pkgconfig:/usr/local/opt/libx11/lib/pkgconfig:/usr/local/opt/libxext/lib/pkgconfig:/usr/local/opt/libxrender/lib/pkgconfig:/usr/local/opt/lzo/lib/pkgconfig:/usr/local/opt/pixman/lib/pkgconfig:/usr/local/opt/cairo/lib/pkgconfig:/usr/local/opt/jpeg/lib/pkgconfig:/usr/local/opt/libtiff/lib/pkgconfig:/usr/local/opt/gdk-pixbuf/lib/pkgconfig:/usr/local/opt/fribidi/lib/pkgconfig:/usr/local/opt/gobject-introspection/lib/pkgconfig:/usr/local/opt/graphite2/lib/pkgconfig:/usr/local/opt/icu4c/lib/pkgconfig:/usr/local/opt/harfbuzz/lib/pkgconfig:/usr/local/opt/pango/lib/pkgconfig:/usr/local/opt/librsvg/lib/pkgconfig:/usr/local/opt/little-cms2/lib/pkgconfig:/usr/local/opt/jansson/lib/pkgconfig:/usr/local/opt/jbig2dec/lib/pkgconfig:/usr/local/opt/libidn/lib/pkgconfig:/usr/local/opt/openjpeg/lib/pkgconfig:/usr/local/opt/aom/lib/pkgconfig:/usr/local/opt/libde265/lib/pkgconfig:/usr/local/opt/x265/lib/pkgconfig:/usr/local/opt/libheif/lib/pkgconfig:/usr/local/opt/liblqr/lib/pkgconfig:/usr/local/opt/imath/lib/pkgconfig:/usr/local/opt/openexr/lib/pkgconfig:/usr/local/opt/webp/lib/pkgconfig:/usr/local/opt/isl/lib/pkgconfig:/usr/local/opt/mpfr/lib/pkgconfig:/usr/local/opt/xorgproto/share/pkgconfig:/usr/local/opt/shared-mime-info/share/pkgconfig
PKG_CONFIG_LIBDIR: /usr/lib/pkgconfig:/usr/local/Homebrew/Library/Homebrew/os/mac/pkgconfig/11
HOMEBREW_GIT: git
HOMEBREW_SDKROOT: /Library/Developer/CommandLineTools/SDKs/MacOSX.sdk
ACLOCAL_PATH: /usr/local/share/aclocal
PATH: /usr/local/opt/gnu-sed/libexec/gnubin:/usr/local/Homebrew/Library/Homebrew/shims/mac/super:/usr/local/opt/autoconf/bin:/usr/local/opt/gnu-sed/bin:/usr/local/opt/pkg-config/bin:/usr/local/opt/texinfo/bin:/usr/local/opt/m4/bin:/usr/local/opt/libtool/bin:/usr/local/opt/guile/bin:/usr/local/opt/gettext/bin:/usr/local/opt/libidn2/bin:/usr/local/opt/libtasn1/bin:/usr/local/opt/nettle/bin:/usr/local/opt/p11-kit/bin:/usr/local/opt/openssl@1.1/bin:/usr/local/opt/libevent/bin:/usr/local/opt/c-ares/bin:/usr/local/opt/jemalloc/bin:/usr/local/opt/nghttp2/bin:/usr/local/opt/gnutls/bin:/usr/local/opt/libpng/bin:/usr/local/opt/freetype/bin:/usr/local/opt/fontconfig/bin:/usr/local/opt/pcre/bin:/usr/local/opt/gdbm/bin:/usr/local/opt/sqlite/bin:/usr/local/opt/xz/bin:/usr/local/opt/python@3.9/bin:/usr/local/opt/glib/bin:/usr/local/opt/cairo/bin:/usr/local/opt/jpeg/bin:/usr/local/opt/libtiff/bin:/usr/local/opt/gdk-pixbuf/bin:/usr/local/opt/fribidi/bin:/usr/local/opt/gobject-introspection/bin:/usr/local/opt/graphite2/bin:/usr/local/opt/icu4c/bin:/usr/local/opt/harfbuzz/bin:/usr/local/opt/pango/bin:/usr/local/opt/librsvg/bin:/usr/local/opt/little-cms2/bin:/usr/local/opt/jbig2dec/bin:/usr/local/opt/libidn/bin:/usr/local/opt/openjpeg/bin:/usr/local/opt/ghostscript/bin:/usr/local/opt/aom/bin:/usr/local/opt/libde265/bin:/usr/local/opt/docbook-xsl/bin:/usr/local/opt/gnu-getopt/bin:/usr/local/opt/xmlto/bin:/usr/local/opt/shared-mime-info/bin:/usr/local/opt/x265/bin:/usr/local/opt/libheif/bin:/usr/local/opt/openexr/bin:/usr/local/opt/webp/bin:/usr/local/opt/imagemagick/bin:/usr/local/opt/gcc/bin:/usr/bin:/bin:/usr/sbin:/sbin
2021-05-25 19:25:45 -0400

./configure
--disable-dependency-tracking
--disable-silent-rules
--enable-locallisppath=/usr/local/share/emacs/site-lisp
--infodir=/usr/local/Cellar/emacs-plus@28/28.0.50/share/info/emacs
--prefix=/usr/local/Cellar/emacs-plus@28/28.0.50
--with-xml2
--with-gnutls
--with-native-compilation
--without-dbus
--with-imagemagick
--with-modules
--with-rsvg
--with-ns
--disable-ns-self-contained

configure: WARNING: unrecognized options: --disable-dependency-tracking
checking for xcrun... xcrun
checking for make... yes
checking for GNU Make... make
checking build system type... x86_64-apple-darwin20.5.0
checking host system type... x86_64-apple-darwin20.5.0
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables...
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether the compiler supports GNU C... yes
checking whether clang accepts -g... yes
checking for clang option to enable C11 features... none needed
checking whether the compiler is clang... yes
checking for compiler option needed when checking for declarations... -Werror=implicit-function-declaration
checking for ar... ar
checking whether clang and cc understand -c and -o together... yes
checking for stdio.h... yes
checking for stdlib.h... yes
checking for string.h... yes
checking for inttypes.h... yes
checking for stdint.h... yes
checking for strings.h... yes
checking for sys/stat.h... yes
checking for sys/types.h... yes
checking for unistd.h... yes
checking for wchar.h... yes
checking for minix/config.h... no
checking for linux/fs.h... no
checking for malloc.h... no
...skipping...
checking for memset_s... yes
checking for fchmodat... yes
checking for lchmod... yes
checking for fcntl... yes
checking for fdopendir... yes
checking for fstatat... yes
checking for fsync... yes
checking for futimens... yes
checking for getrandom... no
checking for gettimeofday... yes
checking for mkostemp... yes
checking for pipe2... no
checking for pselect... yes
checking for pthread_sigmask... yes
checking for readlink... yes
checking for isblank... yes
checking for iswctype... yes
checking for strtoimax... yes
checking for symlink... yes
checking for localtime_r... yes
checking for timegm... yes
checking for utimensat... yes
checking for getdtablesize... yes
checking for futimes... yes
checking for futimesat... no
checking for lutimes... yes
checking for working mmap... yes
checking for main in -lXbsd... no
checking for pthread library... none needed
checking for thread support... yes
checking for librsvg-2.0 >= 2.14.0... yes
checking for MagickWand >= 7... yes
checking for MagickRelinquishMemory... yes
checking for MagickExportImagePixels... yes
checking for MagickMergeImageLayers... yes
checking for MagickAutoOrientImage... yes
checking for getaddrinfo_a in -lanl... no
checking for lgetfilecon in -lselinux... no
checking for gnutls >= 2.12.2... yes
checking for libsystemd >= 222... no
checking for jansson >= 2.7... yes
checking for sys/inotify.h... no
checking for libkqueue... no
checking for library containing kqueue... none required
checking for jpeglib 6b or later... -ljpeg
checking for lcms2... yes
checking for library containing inflateEnd... no
checking for dladdr... yes
checking for dlfunc... no
checking for vfork... yes
checking for fchmod... yes
checking for canonicalize_file_name... no
checking for faccessat... yes
checking for realpath... yes
checking for lstat... yes
checking for readlinkat... yes
checking for explicit_bzero... no
checking for memset_s... yes
checking for fchmodat... yes
checking for lchmod... yes
checking for fcntl... yes
checking for fdopendir... yes
checking for fstatat... yes
checking for fsync... yes
checking for futimens... yes
checking for getrandom... no
checking for gettimeofday... yes
checking for mkostemp... yes
checking for pipe2... no
checking for pselect... yes
checking for pthread_sigmask... yes
checking for readlink... yes
checking for isblank... yes
checking for iswctype... yes
checking for strtoimax... yes
checking for symlink... yes
checking for localtime_r... yes
checking for timegm... yes
checking for utimensat... yes
checking for getdtablesize... yes
checking for futimes... yes
checking for futimesat... no
checking for lutimes... yes
checking for working mmap... yes
checking for main in -lXbsd... no
checking for pthread library... none needed
checking for thread support... yes
checking for librsvg-2.0 >= 2.14.0... yes
checking for MagickWand >= 7... yes
checking for MagickRelinquishMemory... yes
checking for MagickExportImagePixels... yes
checking for MagickMergeImageLayers... yes
checking for MagickAutoOrientImage... yes
checking for getaddrinfo_a in -lanl... no
checking for lgetfilecon in -lselinux... no
checking for gnutls >= 2.12.2... yes
checking for libsystemd >= 222... no
checking for jansson >= 2.7... yes
...skipping...
checking for vfork... yes
checking whether C compiler handles -Wno-switch... yes
checking whether C compiler handles -Wno-pointer-sign... yes
checking whether C compiler handles -Wno-string-plus-int... yes
checking whether C compiler handles -Wno-unknown-attributes... yes
checking whether C compiler handles -Wno-initializer-overrides... yes
checking whether C compiler handles -Wno-tautological-compare... yes
checking whether C compiler handles -Wno-tautological-constant-out-of-range-compare... yes
checking for a BSD-compatible install... /usr/bin/install -c
checking command to symlink files in the same directory... ln -s
checking for install-info... /usr/local/opt/texinfo/bin/install-info
checking for gzip... /usr/bin/gzip
checking for 'find' args to delete a file... -delete
checking for brew... no
checking for -znocombreloc... not needed
checking whether addresses are sanitized... no
checking for library containing sqrt... none required
checking for pkg-config... /usr/local/Homebrew/Library/Homebrew/shims/mac/super/pkg-config
checking pkg-config is at least version 0.9.0... yes
checking for machine/soundcard.h... no
checking for sys/soundcard.h... no
checking for soundcard.h... no
checking for mmsystem.h... no
checking for _oss_ioctl in -lossaudio... no
checking for alsa >= 1.0.0... no
checking for ADDR_NO_RANDOMIZE... no
checking for sys/wait.h that is POSIX.1 compatible... yes
checking for net/if.h... yes
checking for ifaddrs.h... yes
checking for net/if_dl.h... yes
checking for struct ifreq.ifr_flags... yes
checking for struct ifreq.ifr_hwaddr... no
checking for struct ifreq.ifr_netmask... no
checking for struct ifreq.ifr_broadaddr... yes
checking for struct ifreq.ifr_addr... yes
checking for struct ifreq.ifr_addr.sa_len... yes
checking whether gcc understands -MMD -MF... yes
checking for X... libraries , headers
checking for AppKit/AppKit.h... yes
checking for Mac OS X 10.6 or newer... yes
checking whether the compiler supports GNU Objective C... yes
checking whether clang accepts -g... yes
checking if the Objective C compiler supports instancetype... yes
checking if the Objective C compiler defaults to C99... yes
checking whether malloc is Doug Lea style... no
checking for sbrk... yes
checking for getpagesize... yes
checking for __lsan_ignore_object... no
checking for fork... yes
checking for vfork... yes
Your branch is up to date with 'origin/master'.
HEAD is now at f4dc646e0d * lisp/international/mule-cmds.el (mule--ucs-names-group): Better char check.
==> Patching
==> Applying no-frame-refocus-cocoa.patch
patching file src/frame.c
Hunk #1 succeeded at 2096 (offset 40 lines).
==> Applying fix-window-role.patch
patching file src/nsterm.m
Hunk #1 succeeded at 8881 (offset 113 lines).
==> Applying system-appearance.patch
patching file src/frame.h
patching file src/nsfns.m
Hunk #1 succeeded at 1260 (offset 7 lines).
patching file src/nsterm.m
Hunk #1 succeeded at 2173 (offset -21 lines).
Hunk #2 succeeded at 5669 (offset -82 lines).
Hunk #3 succeeded at 5915 (offset -82 lines).
Hunk #4 succeeded at 5965 (offset -82 lines).
Hunk #5 succeeded at 6191 (offset -82 lines).
Hunk #6 succeeded at 9212 (offset 80 lines).
Hunk #7 succeeded at 10298 (offset 284 lines).
==> ImageMagick PKG_CONFIG_PATH:
/usr/local/opt/imagemagick/lib/pkgconfig
==> ./autogen.sh
==> ./configure --enable-locallisppath=/usr/local/share/emacs/site-lisp --infodir=/usr/local/Cellar/emacs-plus@28/28.0.50/share/info/emacs --prefix=/usr/local/Cellar/emacs-plus@28/2
Last 15 lines from /Users/ryan/Library/Logs/Homebrew/emacs-plus@28/02.configure:
checking for MagickAutoOrientImage... yes
checking for getaddrinfo_a in -lanl... no
checking for lgetfilecon in -lselinux... no
checking for gnutls >= 2.12.2... yes
checking for libsystemd >= 222... no
checking for jansson >= 2.7... yes
checking for sys/inotify.h... no
checking for libkqueue... no
checking for library containing kqueue... none required
checking for jpeglib 6b or later... -ljpeg
checking for lcms2... yes
checking for library containing inflateEnd... no
checking for dladdr... yes
checking for dlfunc... no
configure: error: '--with-nativecomp' requires zlib

If reporting this issue please do so at (not Homebrew/brew or Homebrew/core):
  https://github.com/d12frosted/homebrew-emacs-plus/issues

These open issues may also help:
Permission issues with emacs-plus running as a service https://github.com/d12frosted/homebrew-emacs-plus/issues/304
  </pre>
</details>

## Output of `brew config`

<pre>
$ brew config
HOMEBREW_VERSION: 3.1.9-8-g6f1398a
ORIGIN: https://github.com/Homebrew/brew
HEAD: 6f1398a5338d851151041c81562da38b62cceff3
Last commit: 8 hours ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: 854b4eb2ff7f2fcbb28d0f69cf5c6d6b5d04dd21
Core tap last commit: 23 minutes ago
Core tap branch: master
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CASK_OPTS: []
HOMEBREW_DISPLAY: /private/tmp/com.apple.launchd.pQYcmh7dIC/org.xquartz:0
HOMEBREW_EDITOR: vim
HOMEBREW_GITHUB_API_TOKEN: set
HOMEBREW_MAKE_JOBS: 4
Homebrew Ruby: 2.6.3 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: quad-core 64-bit kabylake
Clang: 12.0.5 build 1205
Git: 2.31.1 => /usr/local/bin/git
Curl: 7.64.1 => /usr/bin/curl
macOS: 11.4-x86_64
CLT: 12.5.0.0.1.1617976050
Xcode: 12.5
</pre>


## Output of `brew doctor`

<pre>
$ brew doctor
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry or file an issue; just ignore this. Thanks!

Warning: Some installed kegs have no formulae!
This means they were either deleted or installed with `brew diy`.
You should find replacements for the following formulae:
  python@2
  wine
  unrar
  phantomjs
  chromedriver
  xar-mackyle

Warning: Some installed formulae are deprecated or disabled.
You should find replacements for the following formulae:
  ext4fuse
  ilmbase
  libinfinity
  liblwgeom
  lua@5.1
  makeicns
  proj@7
  py2cairo
  sshfs

Warning: "config" scripts exist outside your system or Homebrew directories.
`./configure` scripts often look for *-config scripts to determine if
software packages are installed, and which additional flags to use when
compiling and linking.

Having additional scripts in your path can confuse software installed via
Homebrew if the config script overrides a system or Homebrew-provided
script of the same name. We found the following "config" scripts:
  /Users/ryan/.pyenv/shims/python2-config
  /Users/ryan/.pyenv/shims/python3.7-config
  /Users/ryan/.pyenv/shims/python2.7-config
  /Users/ryan/.pyenv/shims/python3.7m-config
  /Users/ryan/.pyenv/shims/python-config
  /Users/ryan/.pyenv/shims/python3-config
  /Users/ryan/.pyenv/shims/python3.8-config
  /Users/ryan/.pyenv/shims/audiotools-config

Warning: gettext files detected at a system prefix.
These files can cause compilation and link failures, especially if they
are compiled with improper architectures. Consider removing these files:
  /usr/local/lib/libintl.dylib
  /usr/local/lib/libintl.dylib

Warning: Putting non-prefixed coreutils in your path can cause GMP builds to fail.

Warning: Unbrewed dylibs were found in /usr/local/lib.
If you didn't put them there on purpose they could cause problems when
building Homebrew formulae, and may need to be deleted.

Unexpected dylibs:
  /usr/local/lib/Spyder.dylib
  /usr/local/lib/Spyder3.dylib
  /usr/local/lib/Spyder5.dylib
  /usr/local/lib/dccmtr.dylib
  /usr/local/lib/libCIEHandler.dylib
  /usr/local/lib/libCalibrationHandler.dylib
  /usr/local/lib/libCompression.dylib
  /usr/local/lib/libDeviceManager.dylib
  /usr/local/lib/libErrorHandler.dylib
  /usr/local/lib/libGLEW.1.5.4.dylib
  /usr/local/lib/libICMManager.dylib
  /usr/local/lib/libMCCAppManager.dylib
  /usr/local/lib/libMCCEngineWrapper.dylib
  /usr/local/lib/libMonitorDDCCISDK.dylib
  /usr/local/lib/libProtocolEngine.dylib
  /usr/local/lib/libQM2Lib.dylib
  /usr/local/lib/libSDL-1.2.0.dylib
  /usr/local/lib/libatk-1.0.0.2209.1.dylib
  /usr/local/lib/libdiskmgt.1.dylib
  /usr/local/lib/libgdk-x11-2.0.0.1200.0.dylib
  /usr/local/lib/libgdk_pixbuf-2.0.0.1200.0.dylib
  /usr/local/lib/libgdk_pixbuf_xlib-2.0.0.1200.0.dylib
  /usr/local/lib/libgio-2.0.0.0.0.dylib
  /usr/local/lib/libglib-2.0.0.1600.0.dylib
  /usr/local/lib/libgmodule-2.0.0.1600.0.dylib
  /usr/local/lib/libgobject-2.0.0.1600.0.dylib
  /usr/local/lib/libgthread-2.0.0.1600.0.dylib
  /usr/local/lib/libgtk-x11-2.0.0.1200.0.dylib
  /usr/local/lib/libintl.8.0.2.dylib
  /usr/local/lib/libjpeg.62.dylib
  /usr/local/lib/libnvpair.1.dylib
  /usr/local/lib/libpango-1.0.0.2100.0.dylib
  /usr/local/lib/libpangocairo-1.0.0.2100.0.dylib
  /usr/local/lib/libpangoft2-1.0.0.2100.0.dylib
  /usr/local/lib/libpangox-1.0.0.2100.0.dylib
  /usr/local/lib/libpangoxft-1.0.0.2100.0.dylib
  /usr/local/lib/libpng15.15.dylib
  /usr/local/lib/libportaudio.2.0.0.dylib
  /usr/local/lib/libuutil.1.dylib
  /usr/local/lib/libz.1.2.5.dylib
  /usr/local/lib/libzfs.2.dylib
  /usr/local/lib/libzfs_core.1.dylib
  /usr/local/lib/libzpool.1.dylib

Warning: Unbrewed '.la' files were found in /usr/local/lib.
If you didn't put them there on purpose they could cause problems when
building Homebrew formulae, and may need to be deleted.

Unexpected '.la' files:
  /usr/local/lib/libatk-1.0.la
  /usr/local/lib/libdiskmgt.la
  /usr/local/lib/libgdk-x11-2.0.la
  /usr/local/lib/libgdk_pixbuf-2.0.la
  /usr/local/lib/libgdk_pixbuf_xlib-2.0.la
  /usr/local/lib/libgio-2.0.la
  /usr/local/lib/libglib-2.0.la
  /usr/local/lib/libgmodule-2.0.la
  /usr/local/lib/libgobject-2.0.la
  /usr/local/lib/libgthread-2.0.la
  /usr/local/lib/libgtk-x11-2.0.la
  /usr/local/lib/libintl.la
  /usr/local/lib/libjpeg.la
  /usr/local/lib/libnvpair.la
  /usr/local/lib/libpango-1.0.la
  /usr/local/lib/libpangocairo-1.0.la
  /usr/local/lib/libpangoft2-1.0.la
  /usr/local/lib/libpangox-1.0.la
  /usr/local/lib/libpangoxft-1.0.la
  /usr/local/lib/libportaudio.la
  /usr/local/lib/libuutil.la
  /usr/local/lib/libzfs.la
  /usr/local/lib/libzfs_core.la
  /usr/local/lib/libzpool.la

Warning: Unbrewed static libraries were found in /usr/local/lib.
If you didn't put them there on purpose they could cause problems when
building Homebrew formulae, and may need to be deleted.

Unexpected static libraries:
  /usr/local/lib/libGLEW.a
  /usr/local/lib/libdiskmgt.a
  /usr/local/lib/libintl.a
  /usr/local/lib/libnvpair.a
  /usr/local/lib/libportaudio.a
  /usr/local/lib/libuutil.a
  /usr/local/lib/libz.a
  /usr/local/lib/libzfs.a
  /usr/local/lib/libzfs_core.a
  /usr/local/lib/libzpool.a

Warning: You have unlinked kegs in your Cellar.
Leaving kegs unlinked can lead to build-trouble and cause formulae that depend on
those kegs to fail to run properly once built. Run `brew link` on these:
  syncthing
  gtk+
  python@2
  ghostscript
  lua@5.1
  emacs
  sdl
  gettext
  emacs-plus@28
  glew
  md5sha1sum
  gnupg

Warning: Some installed formulae are not readable:
  phantomjs: uninitialized constant #<Class:0x00007fefa260c6a8>::MinimumMacOSRequirement
</pre>

--- a/Formula/emacs-plus@28.rb
+++ b/Formula/emacs-plus@28.rb
@@ -47,6 +47,7 @@ class EmacsPlusAT28 < EmacsBase
     depends_on "gcc" => :build
     depends_on "gmp" => :build
     depends_on "libjpeg" => :build
+    depends_on "zlib" => :build
   end
 
   #

